requirements.txt

```diff
@@ -2,7 +2,7 @@ aiohttp==3.10.11
 aiofiles==24.1.0
 async-timeout<5.0
 awscli
-asyncio==3.4.3
+#asyncio==3.4.3
```

Comment:
This is used by some scripts and can't be removed.

Reply:
It is standard in any Python >3.3, you should not need it if upgrading Python.